tools/mo/openvino/tools/mo/back/preprocessing.py

```diff
@@ -25,7 +23,7 @@ def update_mean_scale_to_dict(input_nodes: list, mean_scale_val, scale):
         if len(mean_scale_val) != len(input_nodes):
             raise Error('Numbers of inputs and mean/scale values do not match. ' + refer_to_faq_msg(61))
 
-        data = np.copy(mean_scale_val)
+        data = copy(mean_scale_val)
```

Comment:
@pavel-esir, @popovaan, can we figure out why `None` is in the list? Per my understanding, `mean_scale_val` should not contain this value.

Reply:
`None` means that scales or means are not specified in cmd, in this case scales are not specified
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/7de9c7c1-3c19-4103-9da7-63ce08ea469b)

Reply:
Yes, `None` value is expected here when `scale` or `mean` is not set. `data` here has following format:
Dictionary:
`{<node name>: (<mean value>, <scale value>), ...}`
Or list for unnamed inputs:
`[(<mean value>, <scale value>), ...]`

`<mean value>` or `<scale value>` can be `None`.

The error presents only for specific `numpy` version, when `numpy` fails to copy `None`, so it was not visible in CI only in manual runs in particular environment.